torch/_library/utils.py

```diff
@@ -37,3 +37,15 @@ def get_source(stacklevel: int) -> str:
     frame = inspect.getframeinfo(sys._getframe(stacklevel))
     source = f"{frame.filename}:{frame.lineno}"
     return source
+
+
+def parse_namespace(name: str) -> Tuple[str, str]:
+    splits = name.split("::")
+    if len(splits) != 2:
+        raise ValueError(
+            f"Expected `name` to be of the form "
+            f'"namespace::name", but got {name}. '
+            f"Operator names in PyTorch consist of a name "
```

Comment:
it still feels like "Operator names in PyTorch" are still things like sin/cos?
when you actually mean "`name` passed to define/impl_* APIs consist of..."

Reply:
Yup